src/coreclr/nativeaot/Runtime/ThunksMapping.cpp

```diff
@@ -22,7 +22,7 @@
 #define THUNK_SIZE  20
 #elif TARGET_ARM64
 #define THUNK_SIZE  16
-#elif TARGET_LOONGARCH64
+#elif TARGET_LOONGARCH64 || TARGET_RISCV64
 #define THUNK_SIZE  16
```

Comment:
```suggestion
#elif TARGET_LOONGARCH64
#define THUNK_SIZE  16
#elif TARGET_RISCV64
#define THUNK_SIZE  12
```
Since we're using ld's immediate below that's 3 instructions.